packages/drivers/odsp-driver/src/test/odspDriverResolverTest.spec.ts

```diff
@@ -64,16 +66,120 @@ describe("Odsp Driver Resolver", () => {
         const resolvedUrl = await resolver.resolve(request);
 
         // Assert
-        assert.equal(resolvedUrl.fileName, fileName, "FileName should be equal");
-        assert.equal(resolvedUrl.driveId, driveId, "Drive id should be equal");
-        assert.equal(resolvedUrl.siteUrl, siteUrl, "SiteUrl should be equal");
-        assert.equal(resolvedUrl.itemId, "", "Item id should be absent");
-        assert.equal(resolvedUrl.hashedDocumentId, "", "No doc id should be present");
-        assert.equal(resolvedUrl.endpoints.snapshotStorageUrl, "", "Snapshot url should be empty");
+        assert.strictEqual(resolvedUrl.fileName, fileName, "FileName should be equal");
+        assert.strictEqual(resolvedUrl.driveId, driveId, "Drive id should be equal");
+        assert.strictEqual(resolvedUrl.siteUrl, siteUrl, "SiteUrl should be equal");
+        assert.strictEqual(resolvedUrl.itemId, "", "Item id should be absent");
+        assert.strictEqual(resolvedUrl.hashedDocumentId, "", "No doc id should be present");
+        assert.strictEqual(resolvedUrl.endpoints.snapshotStorageUrl, "", "Snapshot url should be empty");
 
         const [, queryString] = request.url.split("?");
         const searchParams = new URLSearchParams(queryString);
-        assert.equal(searchParams.get("path"), filePath, "filePath should match");
-        assert.equal(searchParams.get("driveId"), driveId, "Drive id should match");
+        assert.strictEqual(searchParams.get("path"), filePath, "filePath should match");
+        assert.strictEqual(searchParams.get("driveId"), driveId, "Drive id should match");
+    });
+
+    it("Non-Empty string filePath is resolvable", async () => {
```

Comment:
nit: The description here should form a sentence with the `it` keyword. Something like - it("should resolve non-empty filePath"). Same for other test cases that you have added.